README.md

```diff
@@ -150,6 +129,79 @@ If CallKit has been set up correctly, Tom's device will ring like a phone call,
 
 Note: Due to an issue with ARKit, you cannot start a Remote Streem from a view using the camera https://forums.developer.apple.com/message/411888#411888
 
+#### Roles
+
+Roles dictate which side of the streem your on as in whether you're sharing your video or receiving video. They also affect which tools are available to you. The different roles are:
+
+* `LOCAL_CUSTOMER`: This is the primary role for the customer in a two-way streem
+* `ONSITE_CUSTOMER`: This is the primary role for a customer in an onsite (one-way) streem 
+* `REMOTE_PRO`: This is the primary role for the pro in a two-way streem
+* `ONSITE_PRO`: This is the primary role for a pro in an onsite (one-way) streem
+
+In general if you are starting a remote streem you will want the `LOCAL_CUSTOMER` role. 
+
+### Getting Remote User IDs
+
+In order to start a remote streem, your app will need to supply the `remoteUserId` of the user your app's user is going to streem with. There are two mechanisms available in the SDK for retrieving a `remoteUserId`. The first is through getting a list of recently logged in users. The second is through an invitation. In addition to these two methods, you can maintain your own list of `remoteUserIds` and supply them to your app however you choose. 
```

Comment:
`In order to start a streem with a remote user, your app will need to supply that user's `remoteUserId`.

`recently logged-in users`

Reply:
👍